### ISSUE TITLE
Annotation Files For Custom Dataset :

### ISSUE THREAD

**Issue by lakshaymehra** [frostinassiky/gtad#9]
Hi,

Thanks again for this cool repository!

I see that the data/thumos_annotations folder in the repo contains separate CSV files for the val and test set of THUMOS'14. There is also a thumos_gt.json file inside the evaluation folder.

However, the code inside dataset.py requires a CSV file and a JSON file.

1. Do I also need to generate a JSON file?
2. If I need to generate a JSON file for a custom dataset, what are the contents of that? Do I also need to create separate JSON files for train and validation sets? Can you share an example of the same?

I have generated the HDF5 file containing features for my custom data set. I just need to generate the correct annotation files to start training. So, any help from your side would be really appreciated!

Thank you for your time!

**Comment by lakshaymehra** [comment]
No JSON file is needed for training !

**Comment by mrlihellohorld** [comment]
Hi would u tell me how to generate the HDF5 file containing features for custom data set？thanks

> Hi,
> 
> Thanks again for this cool repository!
> 
> I see that the data/thumos_annotations folder in the repo contains separate CSV files for the val and test set of THUMOS'14. There is also a thumos_gt.json file inside the evaluation folder.
> 
> However, the code inside dataset.py requires a CSV file and a JSON file.
> 
> 1. Do I also need to generate a JSON file?
> 2. If I need to generate a JSON file for a custom dataset, what are the contents of that? Do I also need to create separate JSON files for train and validation sets? Can you share an example of the same?
> 
> I have generated the HDF5 file containing features for my custom data set. I just need to generate the correct annotation files to start training. So, any help from your side would be really appreciated!
> 
> Thank you for your time!

Hi would u tell me how to generate the HDF5 file containing features for custom data set？thanks
